### ISSUE TITLE
stacked colorbar goes to infinity

### ISSUE THREAD

**Issue by jsignell** [python-windrose/windrose#42]
It seems that when the edges of the var bins are specified, the final color should go between the -2 and -1 indexed values, not between -1 and infinity. In the example below, I would expect the last color bin to be between 15 and 20.

```
ax = WindroseAxes.from_ax()
ax.bar(bearings, dists, bins=[0,5,10,15,20], normed=True, opening=0.8, edgecolor='white')
ax.set_legend()
```

![example](https://cloud.githubusercontent.com/assets/4806877/18914127/4b054356-857b-11e6-9695-9e7cf4cb09c1.png)

I would be happy to submit a pull request if you can point me in the right direction.

**Comment by scls19fr** [comment]
Hello,

I suggest to first use latest devel version because I don't think you are running windrose using a french environment (see https://github.com/scls19fr/windrose/issues/17 )
https://github.com/scls19fr/windrose/blob/master/windrose/windrose.py#L189

Why not simply using `bins=[0,5,10,15]` ? so the last color bin will be between 15 and infinity.

but I will be pleased to accept a PR if there is a support for upper bound of last interval both

- infinite (like the current behavior)
- finite (like you are requesting)

I suggest to first create sample data to test visually these 2 behaviors.

I also think you have to think about API (how will switch between these 2 behaviors)
Should it be an other parameter ?
Should it be using only `bins` parameter ?
Ideally a compatible API with current behavior will be better.

Kind regards

**Comment by scls19fr** [comment]
See http://climate.ncsu.edu/windrose.php

![krdu_windrose](https://cloud.githubusercontent.com/assets/22249152/19096455/8535c3e0-8a6a-11e6-9273-718bf5e51ed8.png)

colorbar also goes to infinity

**Comment by ocefpaf** [comment]
This should be solved by #185. Please re-open if you disagree.
